aws-android-sdk-connect/src/main/java/com/amazonaws/services/connect/model/transform/AssignContactCategoryActionDefinitionJsonUnmarshaller.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2010-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.services.connect.model.transform;
+
+import com.amazonaws.services.connect.model.*;
+import com.amazonaws.transform.SimpleTypeJsonUnmarshallers.*;
+import com.amazonaws.transform.*;
+import com.amazonaws.util.json.AwsJsonReader;
+
+
+/**
+ * JSON unmarshaller for POJO AssignContactCategoryActionDefinition
+ */
+class AssignContactCategoryActionDefinitionJsonUnmarshaller implements Unmarshaller<AssignContactCategoryActionDefinition, JsonUnmarshallerContext> {
+
+    public AssignContactCategoryActionDefinition unmarshall(JsonUnmarshallerContext context) throws Exception {
+        AwsJsonReader reader = context.getReader();
+        if (!reader.isContainer()) {
+            reader.skipValue();
+            return null;
+        }
+        AssignContactCategoryActionDefinition assignContactCategoryActionDefinition = new AssignContactCategoryActionDefinition();
+        reader.beginObject();
+        while (reader.hasNext()) {
+            String name = reader.nextName();
+            } else {
+                reader.skipValue();
+            }
+        }
```

Comment:
From my investigation looks like this class has no members and hence we need to render something like:
```
  while (reader.hasNext()) {
      String name = reader.nextName();
      reader.skipValue();
  }
```